### PR TITLE
PR 433 MortgageCalculator

### PR DESCRIPTION
**Pull request by arunaousula9421**
@seanculleton Needed review for Mortgage Calculator

### CODE REVIEW

File: webdriverio-cucumbers/src/pages/mortgageCalculator.page.ts [hunk 1]
```diff
@@ -0,0 +1,98 @@
+import { PropertyDetailsPage } from "../pages/propertyDetails.page";
+var expect = require('chai').expect;
```
Comment (johnbrennandaft) [inline webdriverio-cucumbers/src/pages/mortgageCalculator.page.ts]: mix of syntax here. better to convert this to just `import { expect } from 'chai';` here

File: webdriverio-cucumbers/src/pages/search.page.ts [hunk 2]
```diff
@@ -1,12 +1,11 @@
 var expect = require('chai').expect;
-<<<<<<< HEAD:webdriverio + cucumber/src/pages/search.page.ts
```
Comment (johnbrennandaft) [inline webdriverio-cucumbers/src/pages/search.page.ts]: i guess there was some weirdness with conflict errors that would need to be fixed here

File: webdriverio-cucumbers/src/step-definitions/savedSearchesSteps.ts [hunk 3]
```diff
@@ -4,6 +4,7 @@ import { LoginPage } from "../pages/login.page";
 import { HomePage } from "../pages/home.page";
 import { SearchPage } from "../pages/search.page";
 var expect = require('chai').expect;
```
Comment (johnbrennandaft) [inline webdriverio-cucumbers/src/step-definitions/savedSearchesSteps.ts]: same here regarding imports, assuming they can be imported as normal

Comment (arunaousula9421) [reply webdriverio-cucumbers/src/step-definitions/savedSearchesSteps.ts]: @johnbrennandaft made the changes

File: webdriverio-cucumbers/src/pages/search.page.ts (hunk 2 above)
Comment (arunaousula9421) [reply webdriverio-cucumbers/src/pages/search.page.ts]: @johnbrennandaft there were few conflicts on previous pull request .. so removed them

**Comment by johnbrennandaft** [comment]
fair play on the javascripting Arun, even though this isn't my turf 💃 

One general comment, i think it would be worthwhile to look into adding [Prettier](https://github.com/prettier/prettier) into the project, and add an auto formatting before commits - we do this over in the frontend team and it helps format the code nicely automatically, there is some nitpicks in the PR that I could go into, but for keeping things consistent with spacing etc, would highly recommend it.

File: webdriverio-cucumbers/src/step-definitions/mortgageCalculatorSteps.ts [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+import { MortgageCalculatorPage } from "../pages/mortgageCalculator.page";
+
+
+defineSupportCode(({Given, When, Then}) => {
+    let propertyDetailsPage = new PropertyDetailsPage();
+    let mortgageCalculatorPage = new MortgageCalculatorPage();
+
+    Given(/^Launch Daft and Navigate to PropertyDetails page$/, () => { 
+        propertyDetailsPage
+        .open('galway/houses-for-sale/clifden/the-round-house-errislannan-clifden-galway-451751/');
```
Comment (seanculleton) [inline webdriverio-cucumbers/src/step-definitions/mortgageCalculatorSteps.ts]: @arunaousula9421 Is it ok to hardcode the url here? Will this ad alway be available, and will it exist if you run the test in another test environment?

Comment (arunaousula9421) [reply webdriverio-cucumbers/src/step-definitions/mortgageCalculatorSteps.ts]: @seanculleton This is a staging ad will stay there. We can only see mortgage calculator for the properties which have prices. So had to choose this.

**Comment by arunaousula9421** [comment]
@seanculleton Yeah.. still need to do some many things in here... As it is just the start was looking how it actually works ... Seems to be working good so had to add in more things and make it clean.

**Pull request merged**